src/store/traction/getters.js

```diff
@@ -1,6 +1,4 @@
 const getters = {
-  tagsRequest: (state, getters, rootState) => rootState.api.v1.traction.tags,
-  tractionTags: (state) => state.tractionTags,
```

Comment:
Just to understand - are we not keeping tags in store anymore?

Reply:
Same as above.